x-pack/plugins/graph/public/application.ts

```diff
@@ -12,6 +12,8 @@ import { i18nDirective, i18nFilter, I18nProvider } from '@kbn/i18n/angular';
 import '../../../../webpackShims/ace';
 // required for i18nIdDirective
 import 'angular-sanitize';
+// required for ngRoute
+import 'angular-route';
```

Comment:
See comment about the same file on https://github.com/elastic/kibana/pull/61011#pullrequestreview-381167839. Basically this was imported as a side effect.